h/static/scripts/app.coffee

```diff
@@ -123,6 +123,7 @@ module.exports = angular.module('h', [
 .directive('signinControl', require('./directive/signin-control'))
 .directive('sortDropdown', require('./directive/sort-dropdown'))
 .directive('topBar', require('./directive/top-bar'))
+.directive('hAutofocus', require('./directive/h-autofocus'))
```

Comment:
I'd like to keep this list sorted alphabetically (there are a couple of other items here out of place, I suggest just re-sort the complete list)